schema/policy.go

```diff
@@ -56,22 +56,6 @@ const (
 // AllActions are all possible actions
 var AllActions = []string{ActionCreate, ActionRead, ActionUpdate, ActionDelete}
 
-var adminPolicy = func() *Policy {
```

Comment:
Removed it, as such policy can be defined by adding "tenant_name: <tenant admin name>" to it and specifying it and putting it into external schema.